samples/CommunityToolkit.Maui.Markup.Sample/HotReloadHandler.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+namespace CommunityToolkit.Maui.Markup.Sample;
+
+class HotReloadHandler : ICommunityToolkitHotReloadHandler
+{
+	public async void OnHotReload(IReadOnlyList<Type> types)
```

Comment:
can we change it to Task?

Reply:
We could, but I hesitate to do that for two reasons:
1. This is essentially an event handler for `CommunityToolkitMetadataUpdateHandler.ReloadApplication`
  - It's a fire-and-forget  event
2. Not all implementations will leverage async/await; for example, one could just call `.InvalidateMeasure()` to force a UI update